### ISSUE TITLE
Output of some truffle console commands too dark for Windows Console

### ISSUE THREAD

**Issue by jselbie** [trufflesuite/truffle#3345]
- [ ] I've asked for help in the [Truffle Gitter](http://gitter.im/Consensys/truffle) before filing this issue.

---------------------------

## Issue

What the issue is, in broad strokes.

## Steps to Reproduce

Run:
`truffle console`

Then type:
`web3.eth.getAccounts()`

## Expected Behavior

I can see all the accounts using the default text color of the console

## Actual Results

The accounts are printed in dark blue color that can't be readily seen against a black background

Let me show you a picture: https://imgur.com/a/sKwF7EY

<img src="https://i.imgur.com/CF1QNCB.png"> />

Workaround is to change Windows Console to a different background color

<img src="https://i.imgur.com/q6kSTJg.png" />

## Environment

* Operating System: Windows 10
* Ethereum client: 
* Truffle version (`truffle version`): 5.05
* node version (`node --version`): 12.18.3
* npm version (`npm --version`): 6.14.6

**Comment by fainashalts** [comment]
Hi @jselbie, thanks for bringing this to our attention! Can you tell us what happens if you use Windows terminal instead of console? This may be due to the settings of console.

**Comment by fainashalts** [comment]
Hi @jselbie we can't reproduce this issue - is it possible you have some customized settings on your machine?

**Comment by davidmurdoch** [comment]
@jselbie You can reset your cmd.exe's color settings to their defaults by executing the command `Color 07`. Let me know if that works for you!

**Comment by jselbie** [comment]
@davidmurdoch  - that did not fix the issue.  Still dark text. I will try a clean Windows 10 VM tonight to see I reproduce it there too.

**Comment by eggplantzzz** [comment]
Hey @jselbie, anything to report? Let us know.

**Comment by jselbie** [comment]
I will investigate tonight on a "clean machine".  Thanks for following up.

**Comment by fainashalts** [comment]
Hi @jselbie any updates on this?

**Comment by jselbie** [comment]
I appreciate the follow up and your commitment to resolve these class of
issues.

I installed node+ganache+truffle on a clean Windows 10 VM.  No issue with
colors when I type "web3.eth.getAccounts()" from a truffle console.  The
accounts in the javascrpt were "green".

But on my host machine, I get mixed results depending on which build window
I launch truffle console.  I did figure out that it's now any specific
truffle command. It's the javascript interpreter doing its color syntax
highlighting for any given 'string'.  You can literally type 'string'
(quotes included) into the console and repro it.  Something is screwed up
with the color scheme on my PC.  And I'm at a loss to explain where it's
persisting that setting at.

**Comment by jselbie** [comment]
It basically comes down to when the javascript engine prints a string constant to the console, it highlights it as green or using some specified context id.  If you can point me to which code in the javascript engine is setting colors (and how it chooses which colors to set), I bet I could reverse engineer where it's reading my console color schema and fix that.

**Comment by eggplantzzz** [comment]
@davidmurdoch seemed to have more knowledge about this kind of stuff than I do. Do you know anything?

**Comment by davidmurdoch** [comment]
AFAIK, colors are created by printing a special control code to the terminal/console for each color. The OS (or more maybe more accurately, the shell you are using) interprets these control codes and uses them (or not) to render the output. You can customize the way your shell interprets these codes.

My guess is something on your machine is still changing the way these control codes are interpreted.

**Comment by jselbie** [comment]
**Problem solved.**

I figured the color palette set for cmd.exe and console apps had to be somewhere in the registry, so I just started setting different foreground/background colors via the "properties" dialog (via clicking on the top-left corner).  Then I used ProcessMonitor to observe where it was writing its registry keys when I changed the color scheme.

That let me to discover: `HKEY_CURRENT_USER\Console` is where these settings are stored. And there was one specific for cmd.exe I found:

<img src="https://i.imgur.com/u3PBeU2.png" />

Deleted that `%SystemRoot%_system32_cmd.exe` key which had some suspcious "ColorTable" entries.  Now all is good.

Sorry for the trouble.
